### PR TITLE
Airtable Updates

### PR DESCRIPTION
**Pull request by codygordon**
- [`airtable-python-wrapper` was renamed to `pyairtable` after v1.0](https://github.com/gtalarico/pyairtable/issues/125#issuecomment-891439661). Needed to update this eventually if we want to take advantage of newer features. Seems despite a couple method name changes that it's backwards compatible.
- Added new methods:
  - [update_records](https://pyairtable.readthedocs.io/en/stable/api.html#pyairtable.Table.batch_update)
  - [upsert_records](https://pyairtable.readthedocs.io/en/stable/api.html#pyairtable.Table.batch_upsert)
  - [delete_record](https://pyairtable.readthedocs.io/en/stable/api.html#pyairtable.Table.delete)
  - [delete_records](https://pyairtable.readthedocs.io/en/stable/api.html#pyairtable.Table.batch_delete) 
- Added `typecast` to `insert_record`

### CODE REVIEW

**Comment by codygordon** [comment]
https://github.com/move-coop/parsons/issues/950

**Comment by codygordon** [comment]
also working on addressing [this](https://github.com/move-coop/parsons/issues/758), which I can confirm is still an issue even in the updated lib.

the fix requires fetching the table schema and appending all `fields` to the call, which makes the tests a bit trickier

File: parsons/airtable/airtable.py [hunk 1]
```diff
@@ -159,28 +160,148 @@ def insert_records(self, table, typecast=False):
...
+        `Args:`
+            table: A Parsons Table or list containing the record `id`s to delete.
+        `Returns:`
+            List of dicts with record `id` and `deleted` status
+        """
+
+        if isinstance(table, Table):
+            table = table.to_dicts()
+
+        if any(isinstance(row, dict) for row in table):
```
Comment (shaunagm) [inline parsons/airtable/airtable.py:293]: Can you add a comment here to clarify what's happening on these two lines?

**Review by shaunagm — CHANGES_REQUESTED** [review]
This looks great, just one place that would be good to have a code comment as it's not clear what the code's doing. Once that's added I can merge this

**Review by shaunagm — APPROVED** [review]

**Pull request merged**